lib/tasks/hackathons.rake

```diff
@@ -7,6 +7,7 @@ namespace :hackathons do
     # ```
     # In order to see the output logs
     Hackathon.find_each do |hackathon|
+      next if hackathon.events.where(action: "archived_website").last&.created_at&.after? 1.day.ago
```

Comment:
Maybe we could add `website_last_archived_at?` to the model?

Reply:
this is temporary! it's just in the rake task